### ISSUE TITLE
Add annotator for SyntaxNet

### ISSUE THREAD

**Issue by hiroshinoji** [mynlp/jigg#37]
https://github.com/tensorflow/models/tree/master/syntaxnet

**Comment by hiroshinoji** [comment]
Supported in e1adcda0dbd14a4098e but some problems remain.

One problem in SyntaxNet is that its output (by pretrained model, McParseface) follows the older basic Stanford dependencies (SD), not UD. This may be problematic when considering pipeline with Stanford CoreNLP, in which the dependency parser outputs UD style trees.

In fact, current annotators in CoreNLP that rely on dependency annotations (e.g., coref) fail when the assigned dependencies are not in UD style (probably due to inconsistency of labels).

One solution would be to prepare the model trained from UD treebanks by ourselves...

Another related issue is that current requirements have no mechanism to distinguish different dependency annotation schemes. But this is also the case in CoreNLP, so I will postpone it.

**Comment by hiroshinoji** [comment]
I asked to Stanford group the detail of the current coref system:
https://mailman.stanford.edu/pipermail/java-nlp-user/2016-July/007691.html
and the answer was that in the current release (v. 3.6.0) the intended dependency annotation for coref is still SD, though it will be changed in the next version.

So for temporary, I think coref system can be compatible with the default output of SyntaxNet. The following command is one example to achieve the full pipeline combining SyntaxNet and CoreNLP:

``` bash
jigg.pipeline.Pipeline -annotators "corenlp[tokenize,ssplit],berkeleyparser,syntaxnet,collapseddep,corenlp[lemma,ner,mention,coref]"
```

The POS tagging by `berkeleyparser` is overwritten by `syntaxnet`.  `collapseddep` supplies collapsed and collapsed-cc-processed dependencies based on the output of "basic" dependencies by `syntaxnet`.

Note that this command will not work correctly with the next version of CoreNLP.
